content/blog/2020-09-06-haiku_monthly_activity_report_august_2020.md

```diff
@@ -0,0 +1,106 @@
++++
+type = "blog"
+title = "Haiku Activity Report - August 2020"
+author = "bitigchi"
+date = "2020-09-06 13:00:00+03:00"
+tags = ["haiku", "monthly activity report"]
++++
+
+Welcome to the Haiku Monthly Activity Report for August 2020. This report covers revisions from hrev 54480 to hrev54538.
+
+This month is not active as the last one, but we have exciting improvements in the pipeline, since GSOC 2020 is coming to an end, and our developers are working hard to review GSOC projects at a fast pace.
+
+## Applications
+
+syedsouban, PulkoMandy, and bitigchi worked on adding search engine shortcuts to WebPositive. Now it is possible to use different search engines like DuckDuckGo, Ecosia, Bing etc.
+
+A list of supported search engines and the respective shortcuts can be found below:
+
+- Google (g)
+- Bing (b)
+- Wikipedia (w)
+- DuckDuckGo (d)
+- Baidu (a)
+- Yandex (y)
+- Ecosia (e)
+- Qwant (q)
+
+Example: To search on Ecosia, use: e <search_term>.
+
+bitigchi has committed a fix that allows rebooting immediately when clicked "Reboot" in SoftwareUpdater. Previously it displayed the regular system shutdown prompt, which was not ideal, UX-wise.
+
+Humdinger also improved accessibility on WebPositive by adding a missing tooltip on Settings window.
+
+PulkoMandy and nielx worked on making the Installer more reliable by refining CopyEngine and post-install scripts.
+
+PulkoMandy worked on fixing Pulse issues, now it is possible to resize the window without artifacts and CPU image rescale issues. PulkoMandy also improved battery detection for PowerStatus.
```

Comment:
It is not possible to resize the Pulse window. But the window (and the co,puter chip graphic inside it) now rescales itself to an appropriate size so the CPU information printed there is actually readable.